test/re/frida/MethodTest.java

```diff
@@ -39,6 +40,99 @@ public void replacementPropagatesExceptions() {
         thrown.expectMessage("Already dead");
         badger.die();
     }
+    
+    static private String failString = null;
+    static private void Fail( String msg )
+    {
+      failString = msg;
+    }
+    
+    @Test
+    public void TestNewInterface() {
```

Comment:
Should follow the `camelCase()` style here and below, and also avoid `test` as a prefix since it's already implicit from the class. I tried to follow the examples in the JUnit docs for naming, so in this case perhaps something like `interfaceCannotBeInstantiated()` or something like that.